### DISCUSSION TITLE
Hit modifier keys multiple times

### DISCUSSION THREAD

**Discussion by kanapuli** [manna-harbour/miryoku#243] [Q&A] [answered]
My IDE offers shortcuts, such as 'Shift-Shift,' to display all project files. However, using these shortcuts with my split keyboard presents a challenge as I find it difficult to press the 'Shift' key rapidly twice. When I attempt to hold down 'Shift' on one side of the keyboard and simultaneously press the 'Shift modifier' on the other half, it doesn't function as intended; instead, it types the character in uppercase. The same issue occurs when I attempt a double-tap on 'Shift' with one hand. Do you have any recommendations for resolving this problem? Thank you in advance.

**Comment by manna-harbour** [comment, marked as answer]
To tap mods, use the mods on the layers. E.g. to double tap Shift, hold Nav (left home thumb) and double tap Shift (right home index).
